apps/dashboard/src/components/contract-components/contract-deploy-form/custom-contract.tsx

```diff
@@ -412,11 +437,12 @@ export const CustomContractForm: React.FC<CustomContractFormProps> = ({
           params: {
             name: params.contractMetadata?.name || "",
             contractURI: _contractURI,
-            defaultAdmin: params.deployParams._defaultAdmin,
+            defaultAdmin: params.deployParams._defaultAdmin as string,
```

Comment:
always dangerous to cast like this, produces unknown behavior downstream.

would do default value or throw if you always expect it to be filled. better to fail early

Reply:
Updated.
Only using this for special cases like marketplace or for params that can't have dynamic value.